cmake/partition_manager.cmake

```diff
@@ -522,6 +572,21 @@ else()
 
 endif()
 
+# Ensure that the size of the second slot matches the primary slot when
```

Comment:
With a planned improvement of partition manager this will no longer be needed. Instead it will be possible for the `pm.yml` to specify that the size is shared with `mcuboot_primary` as is the case when the secondary partition is stored in internal flash.

The reason we cannot support this now is that it is not possible to share size with a partition in a different region with the current version of partition manager.

Reply:
great, so there is a task for this ?

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/5487 has the feature.

Yes.